### PR TITLE
Aizad/ Fix toggle derived accordion

### PR DESCRIPTION
**Pull request by aizad-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by ameerul-deriv — DISMISSED** [review]
LGTM

File: src/javascript/app/pages/trade/markets.jsx [hunk 1]
```diff
@@ -533,7 +535,7 @@ class Markets extends React.Component {
                                                     className={classNames('market', {
                                                         'active': subgroup_active,
                                                     })}
-                                                    onClick={toggleAccordion || (subgroup_active ? toggleAccordion : '')}
+                                                    onClick={() => toggleAccordion()}
```
Comment (jim-deriv) [inline src/javascript/app/pages/trade/markets.jsx]:
```suggestion
                                                    onClick={toggleAccordion}
```

Comment (jim-deriv) [reply src/javascript/app/pages/trade/markets.jsx:538]: 😫

Comment (aizad-deriv) [reply src/javascript/app/pages/trade/markets.jsx]: resolved changes 😫

**Review by jim-deriv — APPROVED** [review]

File: src/javascript/app/pages/trade/markets.jsx (hunk 1 above)
Comment (jim-deriv) [reply src/javascript/app/pages/trade/markets.jsx]: 😫

**Review by farabi-deriv — APPROVED** [review]

**Pull request merged**